mkosi/archive.py

```diff
@@ -102,3 +104,19 @@ def make_cpio(src: Path, dst: Path, files: Optional[Iterable[Path]] = None) -> N
         # Make sure tar uses user/group information from the root directory instead of the host.
         options=finalize_passwd_mounts(dst),
     )
+
+
+def normalize_mtime(root: Path, source_date_epoch: Optional[str]) -> None:
+    if source_date_epoch is None:
+        return
+
+    try:
+        mtime_epoch = int(source_date_epoch)
+    except ValueError:
+        die(f"SOURCE_DATE_EPOCH={source_date_epoch} is not a valid integer")
+    if mtime_epoch < 0:
+        die(f"SOURCE_DATE_EPOCH={source_date_epoch} is negative")
+
+    with complete_step("Normalizing mtime"):
```

Comment:
Maybe "Setting mtime to $SOURCE_DATE_EPOCH"? When I saw "normalizing mtime", I had no idea what exactly this means until I read the code.